awx/ui/src/screens/Job/JobOutput/getLineTextHtml.js

```diff
@@ -72,7 +72,7 @@ function replaceStyleAttrs(html) {
 export default function getLineTextHtml({
   created,
   event,
-  start_line,
+  start_line: startLine,
```

Comment:
Why are we changing the name here? Just to make it more JS consistent?

Reply:
yeah, that's it